packages/lib/server/queries/teams/index.ts

```diff
@@ -153,20 +154,20 @@ export async function getTeamWithMembers(args: {
     });
   }
 
-  const team = teams[0];
-  if (!team) return null;
+  const orgOrTeam = teams[0];
```

Comment:
Renamed to better convey that it could be org or a regular team, depending on the params passed to fn.